collect_app/src/androidTest/java/org/odk/collect/android/regression/TriggerWidgetTest.java

```diff
@@ -58,4 +59,16 @@ public void guidanceIcons_ShouldBeCollapsed() {
                 .swipeToEndScreen()
                 .clickFinalize();
     }
+
+    @Test // https://github.com/getodk/collect/issues/5523
+    public void whenTwoTriggerWidgetsPlacedOnOneScreen_keepProperAnswersOnScreenRotation() {
```

Comment:
How would you feel about replacing this with a unit test for `TriggerWidget` that checks that the checkbox gets a new ID each time?